experimental/packages/propagator-aws-xray-lambda/package.json

```
@@ -0,0 +1,68 @@
+{
+  "name": "@opentelemetry/propagator-aws-xray-lambda",
+  "version": "0.49.1",
```

```suggestion
  "version": "0.50.0",
```

Accepting this also needs a package-lock.json update.